lib/network/Images.js

```diff
@@ -23,10 +23,11 @@ Images.prototype.setOnloadCallback = function(callback) {
  * @return {Image} img          The image object
  */
 Images.prototype.load = function(url, brokenUrl) {
```

Comment:
Not at all, my example was the one I was using to test while I dev'ed ... and as you can see it's pretty much a modified version the scalable images example :D